test/unit/job_template_importer_test.rb

```diff
@@ -10,29 +10,26 @@ class JobTemplateImporterTest < ActiveSupport::TestCase
 
     let(:result) do
       name = 'Community Service Restart'
+      metadata = {
+        'model' => 'JobTemplate',
+        'kind' => 'job_template',
+        'name' => 'Service Restart',
+        'job_category' => 'Service Restart',
+        'provider_type' => 'SSH',
+        'feature' => remote_execution_feature.label,
+        'template_inputs' => [
+          { 'name' => 'service_name', 'input_type' => 'user', 'required' => true },
+          { 'name' => 'verbose', 'input_type' => 'user' }
+        ]
+      }
       text = <<-END_TEMPLATE
 <%#
-model: JobTemplateImporter
```

Comment:
We had JobTemplateImporter, while it was replaced with JobTemplate, was it just wrong at the beginning or some additiona change is related to this?